Spigot-API-Patches/0196-Player-elytra-boost-API.patch

```diff
@@ -0,0 +1,35 @@
+From 73e3ac8293d97ebba0c2100a6a988af4e0b8e243 Mon Sep 17 00:00:00 2001
+From: Trigary <trigary0@gmail.com>
+Date: Tue, 14 Apr 2020 12:06:14 +0200
+Subject: [PATCH] Player elytra boost API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 3600b4c8..316f55cc 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1635,6 +1635,21 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     public void openBook(@NotNull ItemStack book);
+ 
++    // Paper start
++    /**
++     * Boost a Player that's {@link #isGliding()} using a {@link Firework}.
++     * The creation of the entity can get cancelled, no boosting is done in that case.
```

Comment:
`If the creation of the entity is cancelled, no boosting is done.` would perhaps sound more natural? I can at least see the current wording confusing non-native english speakers.

Reply:
Sounds good! I shall make that change sometime in the future.